### PR TITLE
Added validation for 'what was affected' page

### PR DESCRIPTION
**Pull request by naomiseminega**
# Description
Added validation logic for when user does not select any options on the  'what was affected' page

Fixes # (1096)

> Please delete options that are not relevant.
- [x] New feature

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made any needed changes to the documentation
- [ ] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

# Required followup work
Add text area to this page for when user selects option 'other' . To be solved by issue 1208

### CODE REVIEW

**Comment by naomiseminega** [comment]
@sastels Somehow the test is not reading the OnSubmit function and is failing...not sure why. Could you have a look when you have a minute :)

**Comment by naomiseminega** [comment]
Test passed. Now fixing Lingui merge error

File: f2/src/forms/__tests__/WhatWasAffectedForm.test.js [hunk 1]
```diff
@@ -38,6 +38,32 @@ describe('<WhatWasAffectedForm />', () => {
...
+            </StateProvider>
+          </I18nProvider>
+        </ThemeProvider>
+      </MemoryRouter>,
+    )
+
+    const checkbox = getByLabelText('whatWasAffectedForm.other')
+    const nextButton = getByRole('button')
+
+    clickOn(checkbox) // Wait for promises to resolve
```
Comment (sastels) [inline f2/src/forms/__tests__/WhatWasAffectedForm.test.js]: nice!

File: f2/src/locales/en.json [hunk 2]
```diff
@@ -21,20 +21,16 @@
   "confirmationPage.backButton": "Back to Your contact details",
   "confirmationPage.businessInfo.business": "Description",
   "confirmationPage.businessInfo.title": "Business",
-  "confirmationPage.businessInfo.title.edit": "Edit Affected Business Assets",
```
Comment (sastels) [inline f2/src/locales/en.json]: 🙁 All these `.edit` lines shouldn't be deleted. There was a bug in a previous version of master that caused this 😢 
Can you add these lines back in?

File: f2/src/locales/fr.json [hunk 3]
```diff
@@ -17,24 +17,19 @@
   "button.cancelReport": "Annuler le rapport",
   "button.edit": "Modifier",
   "confirmationPage.ImpactTitle": "Quelles pourraient être les répercussions?",
-  "confirmationPage.ImpactTitle.edit": "Modifier Quelles sont les répercussions",
```
Comment (sastels) [inline f2/src/locales/fr.json]: These `.edit` lines too

**Pull request merged**